man/systemd.network.xml

```diff
@@ -706,6 +706,12 @@
             <citerefentry project='man-pages'><refentrytitle>inet_pton</refentrytitle><manvolnum>3</manvolnum></citerefentry>.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>Table=</varname></term>
+          <listitem>
+            <para>The table id of the route (an unsigned character).</para>
```

Comment:
"unsigned char" is an implementation detail. You actually give a much better description in the commit message.

```
<term><varname>Table=<replaceable>num</replaceable></varname></term>

<para>The table identifier for the route (a number between 1 and 255).
The table can be retrieved using <command>ip route show table <replaceable>num</replaceable></command>.
</para>
```